src/components/LiquidityWidget/ConnectButton.tsx

```diff
@@ -11,7 +11,7 @@ const ConnectButton = () => {
   const { onPresentConnectModal } = useWalletModal(login, logout, t)
 
   return (
-    <Button onClick={onPresentConnectModal} csx={styles.button}>
+    <Button onClick={onPresentConnectModal} sx={styles.button}>
```

Comment:
I don't see the pragma at the top of this file

Reply:
Done

Reply:
I have added it now